internal/utils/templates/initial_schemas/README.md

```diff
@@ -60,7 +60,7 @@ services:
 
   storage:
     container_name: supabase-storage
-    image: supabase/storage-api:v0.15.0
+    image: supabase/storage-api:v0.16.6
```

Comment:
```suggestion
    image: supabase/storage-api:v0.18.7
```

Reply:
0.16.6 has the latest migrations. 0.18.7 is giving segfaults:
```
2022-08-23T05:41:56: PM2 log: Launching in no daemon mode
2022-08-23T05:41:56: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:56: PM2 log: App [server:0] online
2022-08-23T05:41:57: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:57: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:57: PM2 log: App [server:0] online
2022-08-23T05:41:58: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:58: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:58: PM2 log: App [server:0] online
2022-08-23T05:41:58: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:58: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:58: PM2 log: App [server:0] online
2022-08-23T05:41:59: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:59: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:59: PM2 log: App [server:0] online
2022-08-23T05:41:59: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:59: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:59: PM2 log: App [server:0] online
2022-08-23T05:42:00: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:42:00: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:42:00: PM2 log: App [server:0] online
2022-08-23T05:42:00: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
[...]
```